### ISSUE TITLE
reprojection loss not decreasing

### ISSUE THREAD

**Issue by zcsun52** [Digital-Humans-23/a3#11]
Hi, I already passed the test for ex1, but the reprojection loss is not changing while I ran the code, which doesn't seem right. Are there any procedure I'm missing?

**Comment by zcsun52** [comment]
In other words, as `r_locs.requires_grad=True`, why is `r_locs`  still always 0?

**Comment by zcsun52** [comment]
> 

Why is `r_locs.grad` still None after `r_locs.requires_grad=True`?
